src/main/java/org/broadinstitute/hellbender/cmdline/argumentcollections/IntervalArgumentCollection.java

```diff
@@ -139,6 +140,23 @@ public IntervalMergingRule getIntervalMergingRule() {
         return intervalMergingRule;
     }
 
+    /**
+     * Returns the full set of traversal parameters specified on the command line, including parsed intervals without
+     * merging intervals specified by the user on the command line. This is an advanced use case
+     *
+     * NOTE: this currently does not account for interval exclusion arguments
+     *
+     * @param sequenceDict used to validate intervals
+     * @return the full set of traversal parameters specified on the command line
+     */
+    public List<SimpleInterval> getSpecifiedIntervalsWithoutMerging(final SAMSequenceDictionary sequenceDict ) {
+        if ( ! intervalsSpecified() ) {
+            throw new GATKException("Cannot call getTraversalParameters() without specifying either intervals to include or exclude.");
+        }
+
+        return parseIntervals(new GenomeLocParser(sequenceDict), IntervalMergingRule.NONE, IntervalSetRule.UNION, new ArrayList<String>()).getIntervalsForTraversal();
```

Comment:
The padding is added to the unmerged intervals. I think interval padding is probably correct to be added in this case as it might be relevant for the genelist interval evaluation code elsewhere.